### ISSUE TITLE
[Mod] Commonwealth Rockets

### ISSUE THREAD

**Issue by Tantares** [KSP-CKAN/NetKAN#8386]
Mod info
--------
* Mod name: CRE - Commonwealth Rockets
* Author(s): Beale
* License: Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License
* Homepage: https://forum.kerbalspaceprogram.com/index.php?/topic/177388-110x-cre-stockalike-british-rocketry-2026042020/
* Download: https://github.com/Tantares/CRE/releases

**Comment by HebaruSan** [comment]
This is a duplicate of #8117, and this mod was already indexed in #8118.

**Comment by HebaruSan** [comment]
Hi @Tantares, thanks for the heads-up, but FYI the CKAN infrastructure is set up to handle updates without needing to be notified about them. A bot checks your repo roughly every 30 minutes to see if a new version has been uploaded, and if it has been, the bot generates metadata for that version and adds it to CKAN's database automatically, and if it thinks the compatibility may need to be updated, there's also a phase of manual review during which we can take care of that (see KSP-CKAN/CKAN-meta#2295 and KSP-CKAN/NetKAN#8387).

tl;dr: You can just upload a release and not worry about CKAN!
